numero/numero.cpp

```diff
@@ -193,9 +193,12 @@ int main(int argc, char** argv) {
 		}
 	} while (i < last_group);
 
-	std::partial_sort(holidays.begin(), holidays.begin() + numero::options::output_lines_number, holidays.end());
+	size_t holidays_size = holidays.size();
+	size_t output_lines_number = holidays_size > numero::options::output_lines_number ? numero::options::output_lines_number : holidays_size;
```

Comment:
Along same lines, output_lines_number should be numero::options::output_lines_number_t it seems.

Reply:
Fixed.